content/index.html

```diff
@@ -48,8 +48,11 @@
 </head>
 <body>
     <div class="container">
+       	<!-- part HTML_SETTINGS_HEADER_CUSTOM_SECTION -->
+	{{custom_section}}
+	<!-- part HTML_SETTINGS_HEADER_SETTINGS -->
```

Comment:
Just noticed "setttings header settings" which does not look nice. Let's just call it HTML_SETTINGS_BODY.